### PR TITLE
LPS-118607 create a integration test for updating report when form has nested fields

### PR DESCRIPTION
**Pull request by carolmariaabb**
Related issue: https://issues.liferay.com/browse/LPS-118607

### CODE REVIEW

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/java/com/liferay/dynamic/data/mapping/form/report/web/internal/display/context/DDMFormReportDisplayContext.java [hunk 1]
```diff
@@ -73,9 +75,19 @@ public JSONArray getFieldsJSONArray() throws PortalException {
...
+		Map<String, DDMFormField> ddmFormFieldsMap =
+			ddmForm.getDDMFormFieldsMap(true);
 
-		ddmFormFields.forEach(
+		Set<String> keySet = ddmFormFieldsMap.keySet();
+
+		Stream<String> stream = keySet.stream();
+
+		stream.map(
+			fieldName -> ddmFormFieldsMap.get(fieldName)
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/java/com/liferay/dynamic/data/mapping/form/report/web/internal/display/context/DDMFormReportDisplayContext.java:86]: @marcosapmf, please rename to `ddmFormFieldName`.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/service/test/DDMFormInstanceReportLocalServiceTest.java [hunk 2]
```diff
@@ -70,6 +74,14 @@ public static void setUpClass() {
 		_classNameId = PortalUtil.getClassNameId(DDMFormInstance.class);
 	}
 
+	@Before
+	@Override
+	public void setUp() throws Exception {
+		super.setUp();
+
+		_userId = TestPropsValues.getUserId();
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/service/test/DDMFormInstanceReportLocalServiceTest.java:82]: Well done, @marcosapmf. 🙂

**Pull request closed without merging**